virttest/kernel_interface.py

```diff
@@ -17,13 +17,23 @@ def __init__(self, fs, session=None):
         :param session: ShellSession object of remote or VM
         """
         self.fs = fs
-        if not os.path.isfile(self.fs):
-            raise AttributeError("%s is not available" % self.fs)
         self.func = process.getstatusoutput
         self.session = session
+        if not self._check_isfile():
+            raise AttributeError("%s is not available" % self.fs)
         if self.session:
             self.func = self.session.cmd_status_output
 
+    def _check_isfile(self):
+        """
+        check whether the fs exist in local/remote host/VM
+
```

Comment:
This blank line might be removed.

Reply:
> This blank line might be removed.

Ah, yes, delete the blank line now, thanks!